### PR TITLE
[FIX #6626] add tinymce editor to resource text areas

### LINKED ISSUE #6626: Add tinymce editor to resource metadata
**Is your feature request related to a problem? Please describe.**
I would like to keep formated text and URLs and other HTML elements inside the resource metadata

**Describe the solution you'd like**
Add tinymce widgets to resource form

### PR DESCRIPTION
**Pull request by petrus7**
[FIX #6626] 
this will transform text areas of resource metadata into tinymce plugin. @afabiani i changed the approach to how the plugin is applied to HTML. Previously you set single id as the selector for tinymce. Now it is versatile but requires an explicit declaration of widget in the form. This seems to be a more intuitive approach.

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

For all pull requests:

- [x] Confirm you have read the [contribution guidelines](https://github.com/GeoNode/geonode/blob/master/CONTRIBUTING.md) 
- [x] You have sent a Contribution Licence Agreement (CLA) as necessary (not required for small changes, e.g., fixing typos in the documentation)
- [x] Make sure the first PR targets the master branch, eventual backports will be managed later. This can be ignored if the PR is fixing an issue that only happens in a specific branch, but not in newer ones.

The following are required only for core and extension modules (they are welcomed, but not required, for contrib modules):
- [x] There is a ticket in https://github.com/GeoNode/geonode/issues describing the issue/improvement/feature (a notable exemption is, changes not visible to end-users)
- [x] The issue connected to the PR must have Labels and Milestone assigned
- [x] PR for bug fixes and small new features are presented as a single commit
- [x] Commit message must be in the form "[Fixes #<issue_number>] Title of the Issue"
- [ ] New unit tests have been added covering the changes, unless there is an explanation on why the tests are not necessary/implemented
- [x] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [x] This PR passes the QA checks: flake8 geonode
- [x] Commits changing the **settings**, **UI**, **existing user workflows**, or adding **new functionality**, need to include documentation updates

**Submitting the PR does not require you to check all items, but by the time it gets merged, they should be either satisfied or inapplicable.**

### CODE REVIEW

File: geonode/templates/metadata_base.html [hunk 1]
```diff
@@ -1,234 +1,237 @@
-{% extends "geonode_base.html" %}
```
Comment (afabiani) [inline geonode/templates/metadata_base.html:1]: You completely changed the format of this file, it is impossible to understand what's changed!

File: geonode/templates/metadata_detail.html [hunk 2]
```diff
@@ -1,393 +1,393 @@
-{% extends "geonode_base.html" %}
```
Comment (afabiani) [inline geonode/templates/metadata_detail.html:1]: You completely changed the format of this file, it is impossible to understand what's changed!

**Review by afabiani — CHANGES_REQUESTED** [review]
Two comments, moreover, did you check that his change did not break the attributes HTML efitor?

File: geonode/templates/metadata_base.html [hunk 3]
```diff
@@ -1,234 +1,237 @@
...
-{% load base_tags %}
-{% load guardian_tags %}
-{% load floppyforms %}
-{% load static from staticfiles %}
-
-{% block head %}
-  <script src="{% static "lib/js/jquery.js" %}"></script>
-  <script src="{% static "lib/js/moment-with-locales.js" %}"></script>
-  <script src="{% static "lib/js/bootstrap-datetimepicker.js" %}"></script>
-  <link rel="stylesheet" href="{% static "lib/css/bootstrap-datetimepicker.css" %}" />
```
Comment (petrus7) [inline geonode/templates/metadata_base.html:13]: @afabiani this is the change

File: geonode/templates/metadata_base.html [hunk 4]
```diff
@@ -1,234 +1,237 @@
...
+{% load static from staticfiles %}
+
+{% block head %}
+  <script src="{% static "lib/js/jquery.js" %}"></script>
+  <script src="{% static "lib/js/moment-with-locales.js" %}"></script>
+  <script src="{% static "lib/js/bootstrap-datetimepicker.js" %}"></script>
+  <link rel="stylesheet" href="{% static "lib/css/bootstrap-datetimepicker.css" %}" />
+  <script type="text/javascript" src={% static "tinymce/tinymce.min.js" %}></script>
+  <script type="text/javascript" src={% static "django_tinymce/init_tinymce.js" %}></script>
+  <link rel="stylesheet" type="text/css" id="mce-u0" href= {% static "tinymce/skins/ui/oxide/skin.min.css" %}>
```
Comment (petrus7) [inline geonode/templates/metadata_base.html:16]: @afabiani this is the change. lines 14,15,16 added

Comment (petrus7) [reply geonode/templates/metadata_base.html:16]: The same if or the other file

**Comment by petrus7** [comment]
@afabiani "did you check that his change did not break the attributes HTML editor"
this is the only place i found it is used:
https://github.com/GeoNode/geonode/blob/d679015cf6e0ee3701039f739c17c34b9ddb9bf1/geonode/layers/templates/layers/layer_metadata.html#L109

I'm not sure what is expected result but metadata pages seems to be fine

**Review by afabiani — APPROVED** [review]

**Comment by afabiani** [comment]
Updated docs for this PR [here](https://github.com/GeoNode/documentation/commit/33988d5eb94c77beb63b11d812afb237268fc7d1)

**Pull request merged**